source/admin_manual/doveadm_http_api.rst

```diff
@@ -72,7 +72,7 @@ All commands sent to the API needs to be posted in json format using ``Content-T
                "parameter2": "value",
                "parameter3": "value"
            },
-           "identifier"
+           "tag"
```

Comment:
tag1?

Reply:
Well, it was intentional, but looking at it more closely I see it's "command1" name also, and below there's example with command2/tag2.. So I guess, yeah, changed.